bootstrap/kubeadm/main.go

```diff
@@ -61,6 +61,9 @@ func init() {
 var (
 	metricsAddr              string
 	enableLeaderElection     bool
+	leaseDuration            time.Duration
+	renewDeadline            time.Duration
+	retryPeriod              time.Duration
```

Comment:
should we prefix these and the relevant CLI flags with `leaderElection` to be more explicit?